README.md

```diff
@@ -102,6 +102,44 @@ variables for the test scripts like so:
 
 > `$ make DOCKER_OPTS="-e TEST_ALL_RUBIES=1" test`
 
+In order to run the tests manually in the container-based development, the services
+needed to run them correctly must be started before with:
+
+> `$ script/services start`
+
+Then you can execute the following commands to execute all tests:
+
+> `$ bundle exec rake`
+
+In case you need it/want it, it is possible to manually stop the services by executing:
+> `$ script/services stop`
+
+Another alternative to execute all tests from within the development environment
+where you should not need to manually start/stop the services before is:
+
+> `$ script/test`
```

Comment:
I think this should be the first one to appear.

The documentation for running the tests has become quite complex with all the possible options and I think that the "normal way" (`script/test`) should be more visible.